### ISSUE TITLE
Please add site Peach Puff Translation

### ISSUE THREAD

**Issue by TimeLimit67** [dteviot/WebToEpub#1387]
Greetings! I hope this post finds you well.
Can you add  the site **Peach Puff Translation**?

Here's the TOC page link of the Novel "[After the Male Supporting Role Fell Into My Arms](https://peachpuff.in/after-the-male-supporting-role-fell-into-my-arms/)" translated by **Peach Puff Translation**.

The default parser did not work so I have to make my own parser.

The following are the values/codes I utilized to fill in the required fields for the default parser. Honestly, I don't have any idea which of these did not work since I did not encounter any problem when I made one until now.
* **URL of first chapter:** `https://peachpuff.in/fima-ch1-rebirth/`
* **CSS selector for element holding content to put into EPUB:** `article#post-6.post.type-post.status-publish.format-standard.hentry.category-after-the-male-supporting-role-fell-into-my-arms`
* **CSS selector for element holding Title of Chapter:**
* **CSS selector for element(s) to remove:**`div.select-wrapper, div.code-block.code-block-1`

![image](https://github.com/user-attachments/assets/24252b60-d384-438a-bf5e-902cc11de32f)

I have zero developer skills thus I have no idea how to make my own parser. I just settle with making a default parser which most of the time worked. I really wasn't able to extract chapters from Peach Puff Translation even though I am sure I entered the values/codes correctly, no more no less.

~

When I clicked the test button, there's no problem with the output. It is correct with Chapter Title and Chapter Content so I thought it's good already.
![image](https://github.com/user-attachments/assets/5e625c5e-82d4-4282-9252-549887079014)

I clicked the apply button once I checked there's no unnecessary gibberish in the content, The result also does not have any problem. WebToEpub loads correctly with the complete chapters and some other unimportant links that I can just deselect.
![image](https://github.com/user-attachments/assets/9de1b15a-9369-492e-a139-ece7559af90b)
 
However, when I start packing the epub, I get the following errors:
Chapter 1 has green check mark - meaning this chapter was downloaded. But when it reaches chapter 2, chapter 3, and the succeeding chapters, they just won't be downloaded. Repeated error messages pop-up:

![image](https://github.com/user-attachments/assets/ae98d00f-95a5-40d4-9524-a3934744a150)
![image](https://github.com/user-attachments/assets/398bf569-47e6-4ff4-b7ce-67b61d81f0ba)
~
![image](https://github.com/user-attachments/assets/871d5e43-70e5-49ce-8297-ed71119ae53b)
![image](https://github.com/user-attachments/assets/ad196a87-8cca-420a-be42-5be95b1eb2a1)
![image](https://github.com/user-attachments/assets/5174fef6-9299-4682-8b6e-49fab2dc1a10)

In other words, it's fine at the beginning but the succeeding chapters can't be downloaded at all.

Please let me know if there's any mistake. Thanks in advance and have a great day ahead!

**Comment by dteviot** [comment]
@TimeLimit67 

The reason it didn't work is your selector for **CSS selector for element holding content to put into EPUB** was to specific.
This part 
> article#post-6

means look for an &lt;article&gt; with an **id** of "post-6".
Which it is on the first chapter.  On the second chapter, the id is "post-9".
If you look at the page, there is only one &lt;article&gt; element.  So, you just needed **article** as your selector.

Anyway:  

I've added the site.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://peachpuff.in/after-the-male-supporting-role-fell-into-my-arms/, chapters 1, 2, 3

For my notes:  35 minutes work  (21 minutes to write a parser, then 14 minutes to actually READ the incident and realize the problem.)

**Comment by dteviot** [comment]
Updated version (0.0.0.167) has been submitted to Firefox and Chrome stores.
Firefox version is available now.
Chrome might be available in a few hours to 21 days.
